DailySelfie/app/src/main/java/com/teamagam/dailyselfie/MainActivity.java

```diff
@@ -135,4 +137,12 @@ private PictureInfo pictureInfoFromFile(File file) {
         return pictureInfo;
     }
 
+    private void enlargePicture(String path) {
+        final Dialog enlargePictureDialog = new Dialog(this);
```

Comment:
I'd create a class that extends dialog that handles all that is necessary for the image displaying.
You could also add some static method to it that initializes a new Dialog given a path, shows it, and execute it's appropriate async-task.